widgetry/src/widgets/autocomplete.rs

```diff
@@ -90,11 +96,14 @@ impl<T: 'static + Clone> WidgetImpl for Autocomplete<T> {
     fn event(&mut self, ctx: &mut EventCtx, output: &mut WidgetOutput) {
         self.tb.event(ctx, output);
         if self.tb.get_line() != self.current_line {
+            // This will return Outcome::Changed to the caller with a dummy ID for the textbox
             self.current_line = self.tb.get_line();
             self.recalc_menu(ctx);
             output.redo_layout = true;
         } else {
-            // Don't let the menu fill out the real outcome.
+            // Don't let the menu fill out the real outcome. Should we use Outcome::Changed to
```

Comment:
Autocomplete has a pretty weird way of interacting with it today; see map_gui/src/tools/navigate.rs